cet/wrss.c

```diff
@@ -159,15 +158,19 @@ int main(int argc, char *argv[])
 {
 	unsigned long *current_shstk;
 
+	if (ARCH_PRCTL(ARCH_CET_ENABLE, CET_SHSTK)) {
+		printf("[SKIP]\tCould not enable Shadow stack.\n");
+		return 1;
+	}
+	printf("[PASS]\tEnable SHSTK successfully\n");
+
 	if (!_get_ssp()) {
 		printf("[SKIP]\tSHSTK disabled, could not get shstk addr.\n");
 		return 1;
 	}
 
 	sethandler(SIGILL, sigill_receive_expected, 0);
-	current_shstk = (unsigned long *)_get_ssp();
```

Comment:
Hi PengFei, by removing this func call _get_ssp, the local variable `current_shstk` would have a random value when calling `write_shstk`. Did you really mean that ?

Reply:
Yes, I should not remove this line, it's a mistake.

WIll add it back.

Thanks!

Reply:
Done, added it back into wrss.c.

Thanks!